test/src/org/labkey/test/tests/targetedms/TargetedMSQCGuideSetTest.java

```diff
@@ -480,7 +494,7 @@ private void verifyGuideSet5Stats(GuideSet gs) throws IOException, CommandExcept
 
     private void verifyGuideSetSmallMoleculeStats(GuideSet gs) throws IOException, CommandException
     {
-        String precursor = "C16";
+        String precursor = "C16,";
```

Comment:
@labkey-jeckels had to do this to filter out series labels having C16:0, C16:1, C16-H ... Let me know if you have another suggestion